### DISCUSSION TITLE
How to make a delete forward button ?

### DISCUSSION THREAD

**Discussion by DungnvKaopiz** [MathKeyboardEngine/MathKeyboardEngine#5] [Q&A] [answered]
I can only find your deleteCurrent(), how do i make a delete forward button ?

**Comment by SymboLinker** [comment, marked as answer]
That button should first invoke `moveRight()` and then `deleteCurrent()`.

**Reply by DungnvKaopiz** [reply]
lol, nice hack

**Comment by SymboLinker** [comment]
Hello @DungnvKaopiz,

MathKeyboardEngine version 1.0.0 contains a new function - `deleteRight()`. So, custom hacks for that are not needed anymore. I hope the behaviour of it is as expected - or better :)

Because of the introduction of `deleteRight()`, the function `deleteCurrent()` (v0.x) has been renamed to `deleteLeft()` (v1.0.0).

Kind regards,
SymboLinker
